### ISSUE TITLE
Non default build folder.

### ISSUE THREAD

**Issue by swojtasiak** [mesonbuild/vscode-meson#74]
If build folder is other than "builddir", "Could not fetch targets. See Meson Build output tab for more info." information is shown while a mason task is run. The "Mason Build" output shows "false" only, instead of an information about the issue.

**Comment by Ongy** [comment]
Did you configure the project first? Either via the command inside vscode, or by calling meson directly?

**Comment by Ongy** [comment]
That is, configured into the configured build directory.
The error messages aren't too helpful when the directory doesn't exist, but it works for me when it does.

**Comment by swojtasiak** [comment]
Yes as far as I remember the project was configured using mason directly before it was opened in vscode.

**Comment by MGilleronFJ** [comment]
I think I had this issue too, I had already done `meson setup build` before (I prefer `build` to `builddir`). Then I opened VSCode and the Meson extension didn't seem to find my already-configured build directory. I restarted again and noticed it prompted a notification asking if I want to configure the project. Before clicking a button, I went to settings of the extension, saw it had a `build folder` setting, which I changed to `build` to match my folder. Then I clicked `Yes` to configure the build. It took a while, and indeed it had created another `builddir` next to my already-configured `build` folder... (however this time I could see targets). I closed VSCode, deleted that extra build folder, launched VSCode again and it prompted again if I want to configure the build folder. Clicked `Yes`, and finally it detected my `build` folder.
